api_utils/except_err.py

```
@@ -0,0 +1,60 @@
+    """
+    err = str(err)
+    nn = inspect.stack()[2][3]
+    return f"\ndef {nn}(): {err}"
+
+
+common_errors = [
+    "the JSON object must be str, bytes or bytearray, not NoneType",
+]
+
+
+def exception_err(e, text=""):
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)